packages/eventual-send/src/local.js

```diff
@@ -1,38 +1,49 @@
-const q = JSON.stringify;
-
 const ntypeof = specimen => (specimen === null ? 'null' : typeof specimen);
 
 export const localApplyFunction = (t, args) => {
-  if (!(t instanceof Function)) {
-    const ftype = ntypeof(t);
-    throw TypeError(
-      `Cannot invoke target as a function; typeof target is ${q(ftype)}`,
-    );
-  }
+  const { details: X, quote: q } = assert;
```

Comment:
Since you just added this anew, any reason you nested it here (and repeated below) rather than immediately putting it at the top of the file right below imports?

Reply:
Caught me with my pants down.  It's better now.